### PR TITLE
docs: 1.17.1 kickoff

### PR DESCRIPTION
**Pull request by Shikugawa**
Signed-off-by: Shikugawa <rei@tetrate.io>

For an explanation of how to fill out the fields, please see the relevant section
in [PULL_REQUESTS.md](https://github.com/envoyproxy/envoy/blob/main/PULL_REQUESTS.md)

Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/master/api/review_checklist.md):]

### CODE REVIEW

File: ci/filter_example_setup.sh [hunk 1]
```diff
@@ -24,6 +24,7 @@ sed -e "s|{ENVOY_SRCDIR}|${ENVOY_SRCDIR}|" "${ENVOY_SRCDIR}"/ci/WORKSPACE.filter
 
 mkdir -p "${ENVOY_FILTER_EXAMPLE_SRCDIR}"/bazel
 ln -sf "${ENVOY_SRCDIR}"/bazel/get_workspace_status "${ENVOY_FILTER_EXAMPLE_SRCDIR}"/bazel/
+rm -r "${ENVOY_SRCDIR}"/.bazelversion && cp -f "${ENVOY_SRCDIR}"/.bazelversion "${ENVOY_FILTER_EXAMPLE_SRCDIR}"/
```
Comment (lizan) [inline ci/filter_example_setup.sh:27]:
```suggestion
rm -r "${ENVOY_FILTER_EXAMPLE_SRCDIR}"/.bazelversion && cp -f "${ENVOY_SRCDIR}"/.bazelversion "${ENVOY_FILTER_EXAMPLE_SRCDIR}"/
```

**Comment by Shikugawa** [comment]
@moderation cherry-picked kafka dependency update to this. Could you take a look?

**Comment by PiotrSikora** [comment]
I think you need to cherry-pick #14870 to fix examples.

**Pull request merged**